packages/launch/src/focused-launch/index.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { Title } from '@automattic/onboarding';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { TextControl } from '@wordpress/components';
+import * as React from 'react';
+import DomainPicker from '@automattic/domain-picker';
+import { useSite, useDomainSearch } from '../hooks';
+
+import './styles.scss';
+
+const FocusedLaunch: React.FunctionComponent = () => {
+	const { getSite } = useSelect( ( select ) => select( 'core' ) ) as any;
+	const title = getSite()?.title;
+
+	const site = useSite();
+
+	const [ siteTitle, setSiteTitle ] = React.useState( title );
+	const [ siteDomainName ] = React.useState( site.currentDomainName );
+	const domainSearch = useDomainSearch();
+
+	return (
+		<>
+			<div className="focused-launch__section">
+				<Title>{ __( "You're almost there", 'full-site-editing' ) }</Title>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **33** times:
`translate( 'You're almost there!' )` *ES Score: 11*